tests/bud.bats

```diff
@@ -3902,3 +3899,16 @@ _EOF
   expect_output --substring "hello2"
   run_buildah rmi -f testbud
 }
+
+@test "bud with network names" {
+  skip_if_no_runtime
+
+  _prefetch alpine
+
+  run_buildah 125 bud --signature-policy ${TESTSDIR}/policy.json --network nontexists ${TESTSDIR}/bud/network
+  expect_output --substring "network not found"
+
+  run_buildah bud --signature-policy ${TESTSDIR}/policy.json --network podman ${TESTSDIR}/bud/network
```

Comment:
Where does the configuration for the network with this name come from?

Reply:
podman is the default network name (can be changed in cotnainers.conf). If it does not exists on disk the default will always be created in memory, this happens in the libnetwork backend.